static/sass/_pattern_strip.scss

```diff
@@ -24,6 +24,7 @@
   .p-strip--suru {
     @extend %vf-strip;
     background-blend-mode: multiply, multiply, normal, normal;
+    background-color: #772953;
```

Comment:
Just checked this out on Chrome mobile, looks good, but it makes the suru on large screens look a lot darker - is that intended?

Reply:
perhaps do it only in a media query